report/Report.cpp

```diff
@@ -273,6 +285,26 @@ bool ReportPass::runOnFunction(Function &F) {
   LLVMContext &Ctx = F.getContext();
   std::string file_name = F.getParent()->getSourceFileName();
 
+  int demangle_status;
+  char *demangled_fname =
+      abi::__cxa_demangle(fname.c_str(), 0, 0, &demangle_status);
```

Comment:
since the function name is already demangled here, should I just dump the demangled name to JSON? @ethanbwang